app/api/orders.py

```diff
@@ -68,14 +68,14 @@ def is_payment_valid(order, mode):
         return (order.paid_via == 'paypal') and order.transaction_id
 
 
-def check_billing_info(data):
+def check_billing_info(data, order):
```

Comment:
Where is order used?

Reply:
It was used when I split ed into two function.